pynio/block.py

```diff
@@ -118,3 +118,16 @@ def copy(self, name, instance=None):
         out._name = name
         out.config['name'] = name
         return out
+
+    def in_use(self, instance=None):
+        '''returns a dictionary of services that use this block
+
+        If instance is None (default) uses it's own instance
+        '''
+        if instance is None:
+            instance = self._instance
+
+        name = self.name
+        return {sname: service for (sname, service) in instance.services.items()
```

Comment:
A lis tis easier to work with. 

It seems to me that when getting the services, you'll probably want to iterate over them and do something with each one as opposed to picking out one by name like I often do with instance.services. With this list you can do things like:
[s.name for s in b.in_use()]
[s.remove_block(b) for s in b.in_use()]
etc...

I find it unlikely that you would want to do:
b.in_use()['service_name']
which is the perk of a dictionary.